apidoc/Titanium/Network/HTTPClient.yml

```diff
@@ -311,6 +311,16 @@ properties:
     permission: read-only
     platforms: [android]
     
+  - name: responseHeaders
+    summary: Returns all the response headers returned with the request
+    description: |
+        Contains a dictionary of responseheaders only after the http call has finished.
+
+        See also: [getResponseHeader](Titanium.Network.HTTPClient.getResponseHeader).
+    type: Array<Object>
```

Comment:
The type still looks invalid. The code says it's a Dictionary [here](https://github.com/appcelerator/titanium_mobile/blob/master/iphone/Classes/TiNetworkHTTPClientProxy.m#L536) and [here](https://github.com/appcelerator/titanium_mobile/blob/master/iphone/Classes/TiNetworkHTTPClientProxy.h#L46). So it has to be: `type: Dictionary` unless you really receive an array.